momento-sdk/src/main/java/momento/sdk/ScsControlGrpcStubsManager.java

```diff
@@ -26,16 +29,24 @@ final class ScsControlGrpcStubsManager implements AutoCloseable {
 
   private final ScsControlGrpc.ScsControlFutureStub futureStub;
 
-  ScsControlGrpcStubsManager(@Nonnull CredentialProvider credentialProvider) {
-    this.channel = setupConnection(credentialProvider);
+  ScsControlGrpcStubsManager(
+      @Nonnull CredentialProvider credentialProvider, Configuration configuration) {
+    this.channel = setupConnection(credentialProvider, configuration);
     this.futureStub = ScsControlGrpc.newFutureStub(channel);
   }
 
-  private static ManagedChannel setupConnection(CredentialProvider credentialProvider) {
-    final NettyChannelBuilder channelBuilder =
+  private static ManagedChannel setupConnection(
+      CredentialProvider credentialProvider, Configuration configuration) {
+    NettyChannelBuilder channelBuilder =
         NettyChannelBuilder.forAddress(credentialProvider.getControlEndpoint(), 443);
-    channelBuilder.useTransportSecurity();
-    channelBuilder.disableRetry();
+
+    // Override grpc config to disable keepalive for control clients
+    final GrpcConfiguration controlConfig =
+        configuration.getTransportStrategy().getGrpcConfiguration().withKeepAliveDisabled();
+
+    // set additional channel options (message size, keepalive, auth, etc)
+    channelBuilder = GrpcChannelOptions.GrpcOptionsFromGrpcConfig(controlConfig, channelBuilder);
```

Comment:
if you change this method sig the way I suggested below, I think this var will be able to go back to being `final`.